HSP2tools/readWDM.py

```diff
@@ -149,137 +171,199 @@ def readWDM(wdmfile, hdffile, jupyterlab=True):
             summary.append(data)
             summaryindx.append(dsname[11:])
 
+            self.dfsummary = pd.DataFrame(summary, index=summaryindx, columns=columns)
+            #store.put('TIMESERIES/SUMMARY',dfsummary, format='t', data_columns=True)
+        #return dfsummary
+
+# export data to HD5 file
+    def WriteHD5(self, hdffile, compress_output=True):
```

Comment:
Good call on isolating the write process! This will eventually be replaced with the IO abstraction class so it is great that you are already isolating the output method.